WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/ContactSupportScreen.java

```diff
@@ -0,0 +1,119 @@
+package org.wordpress.android.e2e.pages;
+
+import androidx.test.espresso.Espresso;
+import androidx.test.espresso.ViewInteraction;
+import androidx.test.espresso.action.ViewActions;
+
+import org.wordpress.android.R;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.hasDescendant;
+import static androidx.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.isEnabled;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.anyOf;
+import static org.hamcrest.Matchers.not;
+import static org.wordpress.android.support.WPSupportUtils.populateTextField;
+import static org.wordpress.android.support.WPSupportUtils.sleep;
+import static org.wordpress.android.support.WPSupportUtils.waitForElementToBeDisplayed;
+import static org.wordpress.android.support.WPSupportUtils.waitForElementToBeDisplayedWithoutFailure;
+
+public class ContactSupportScreen {
+    // "Contact Support" screen looks differently depending on
+    // "gradle.properties" content (default or from Mobile Secrets).
+    // But the elements tree always contains all elements, some are
+    // just hidden. Locators below attempt to support both variants.
+    static ViewInteraction textInput = onView(allOf(
+            isCompletelyDisplayed(),
+            anyOf(
+                    withId(R.id.message_composer_input_text),
+                    withId(R.id.request_message_field)
+            )
+    ));
```

Comment:
This translates to: find an element that is both completely displayed and has either `R.id.message_composer_input_text` or `R.id.request_message_field` ID.

In fact, both of these elements are present when the screen is opened, but only one is visible.